rust/src/wallet.rs

```diff
@@ -27,6 +27,7 @@ pub enum Network {
 pub struct Wallet {
     blockchain: ElectrumBlockchain,
     wallet: bdk::Wallet<MemoryDatabase>,
+    seed_phrase: Vec<String>,
```

Comment:
are you planning to address my earlier comments from the initial seed review? I would prefer if we didn't need to keep strings around, so we don't accidentally print them out in tracing logs etc.... (can be done as a follow-up, no need to block this PR)

Reply:
Yes 🙂 https://github.com/itchysats/10101/pull/102/commits/b21679b29dcc501066a810b0267aded26b755ab1